pkg/images/consts.go

```diff
@@ -20,4 +20,6 @@ const (
 	SchedulerPluginSchedulerDefaultImage  = "k8s.gcr.io/scheduler-plugins/kube-scheduler:v0.19.9"
 	SchedulerPluginControllerDefaultImage = "k8s.gcr.io/scheduler-plugins/controller:v0.19.9"
 	ResourceTopologyExporterDefaultImage  = "quay.io/openshift-kni/resource-topology-exporter:v0.0.19"
+	// TODO replace this image with some image from k8stopologyawareschedwg quay repo
+	NodeFeatureDiscoveryDefaultImage = "quay.io/swsehgal/node-feature-discovery:v0.10.0-devel-36-g5b0761f3"
```

Comment:
better not to use developers' repo unless we really don't have options. We need to find out a flow to build images, or to use u/s ones.

Reply:
We can have a release flow as RTE has: https://github.com/k8stopologyawareschedwg/resource-topology-exporter/blob/master/.github/workflows/release.yml

Reply:
yes, that'sa a good starting point, but it needs few extra steps - and to be setup in our fork :)